app/src/components/market/common/additional_market_data/index.tsx

```diff
@@ -64,10 +64,27 @@ interface Props extends DOMAttributes<HTMLDivElement> {
 
 export const AdditionalMarketData: React.FC<Props> = props => {
   const { arbitrator, category, id, oracle } = props
-
+  const [connected, setConnected] = useState<boolean>()
   const windowObj: any = window
+
+  useEffect(() => {
+    const fetchUserAccounts = async () => {
+      const account = await windowObj.ethereum.request({ method: 'eth_accounts' })
+
+      setConnected(account.length !== 0)
+
+      windowObj.ethereum.on('accountsChanged', (accounts: string[]) => {
+        setConnected(accounts.length !== 0)
+      })
+    }
+    if (windowObj.ethereum) {
+      fetchUserAccounts()
+    }
+  }, [windowObj.ethereum])
   const realitioBaseUrl =
-    windowObj.ethereum && windowObj.ethereum.isMetaMask ? 'https://reality.eth' : 'https://reality.eth.link'
+    windowObj.ethereum && windowObj.ethereum.isMetaMask && connected
```

Comment:
Should be ok to remove `windowObj.ethereum && windowObj.ethereum.isMetaMask`, and just check that it's `connected`.

Reply:
i implemented that in this new commit!